### PR TITLE
Update the used QGIS version

### PR DESCRIPTION
**Pull request by sbrunner**

### CODE REVIEW

File: .github/workflows/qgis.yaml [hunk 1]
```diff
@@ -15,12 +15,10 @@ jobs:
       fail-fast: false
       matrix:
         version:
-          # Old LTR
-          - '3.4'
           # LTR
           - '3.10'
           # Last
-          - '3.14'
+          - '3.16'
```
Comment (arnaud-morvan) [inline .github/workflows/qgis.yaml:21]: Note that 3.16 is also an LTR

**Review by arnaud-morvan — APPROVED** [review]
Thanks

File: .github/workflows/qgis.yaml (hunk 1 above)
Comment (sbrunner) [reply .github/workflows/qgis.yaml:21]: +1, thanks :-)

File: .github/workflows/qgis.yaml [hunk 2]
```diff
@@ -16,11 +16,9 @@ jobs:
       matrix:
         version:
           # Old LTR
-          - '3.4'
-          # LTR
           - '3.10'
-          # Last
-          - '3.14'
+          # Last LTR
+          - '3.16'
```
Comment (ger-benjamin) [inline .github/workflows/qgis.yaml:21]: The last LTR is still the 3.10 no ?
3.16 is only a standard release

Comment (arnaud-morvan) [reply .github/workflows/qgis.yaml:21]: @ger-benjamin : see https://www.qgis.org/en/site/getinvolved/development/roadmap.html#release-schedule

Comment (ger-benjamin) [reply .github/workflows/qgis.yaml:21]: Thanks you are right.
They are not up to date on their whole website. Sometimes (like on the download page or simply the flag at the upper-left corner) they still say the 3.10 is the latest LTR.

Comment (sbrunner) [reply .github/workflows/qgis.yaml:21]: @ger-benjamin strictly your right that the version 3.16 is the last LTR only from 2021-02-19...

**Pull request merged**